src/Domain/Domain.hpp

```diff
@@ -89,13 +89,14 @@ class Domain {
    * user must provide the PairOfFaces corresponding to the rotated corners.
    *
    */
-  Domain(const Index<VolumeDim>& domain_extents,
-         const std::array<std::vector<double>, VolumeDim>& block_demarcations,
-         const std::vector<Index<VolumeDim>>& block_indices_to_exclude = {},
-         const std::vector<OrientationMap<VolumeDim>>&
-             orientations_of_all_blocks = {},
-         const std::vector<PairOfFaces>& identifications = {},
-         bool use_equiangular_map = false);
+  static Domain multicube_domain(
```

Comment:
I would move this into DomainHelpers @nilsdeppe do you care?

Reply:
Yea, I would do the same since as the name implies this is for a specific domain.